src/sdk/main/include/TokenUpdateNftsTransaction.h

```diff
@@ -0,0 +1,171 @@
+   */
+  [[nodiscard]] inline TokenId getTokenId() const { return mTokenId; }
+
+  /**
+   * Get the serials for the token.
+   *
+   * @return The serials for the token. Returns empty vector if serials are not set.
+   */
+  [[nodiscard]] inline std::vector<uint64_t> getSerials() const { return mSerials; }
+
+  /**
+   * Get the new metadata for the tokens.
+   *
+   * @return The new metadata for the tokens. Returns uninitialized if no new symbol has been set.
+   */
+  [[nodiscard]] inline std::vector<std::byte> getMetadata() const { return mMetadata; }
+
+private:
+  friend class WrappedTransaction;
+
+  /**
+   * Derived from Executable. Submit a Transaction protobuf object which contains this TokenUpdateNftsTransaction's data
+   * to a Node.
+   *
+   * @param request  The Transaction protobuf object to submit.
+   * @param node     The Node to which to submit the request.
+   * @param deadline The deadline for submitting the request.
+   * @param response Pointer to the ProtoResponseType object that gRPC should populate with the response information
+   *                 from the gRPC server.
+   * @return The gRPC status of the submission.
+   */
+  [[nodiscard]] grpc::Status submitRequest(const proto::Transaction& request,
+                                           const std::shared_ptr<internal::Node>& node,
+                                           const std::chrono::system_clock::time_point& deadline,
+                                           proto::TransactionResponse* response) const override;
+
+  /**
+   * Derived from Transaction. Verify that all the checksums in this TokenUpdateNftsTransaction are valid.
+   *
+   * @param client The Client that should be used to validate the checksums.
+   * @throws BadEntityException This TokenUpdateNftsTransaction's checksums are not valid.
+   */
+  void validateChecksums(const Client& client) const override;
+
+  /**
+   * Derived from Transaction. Build and add the TokenUpdateNftsTransaction protobuf representation to the Transaction
+   * protobuf object.
+   *
+   * @param body The TransactionBody protobuf object being built.
+   */
+  void addToBody(proto::TransactionBody& body) const override;
+
+  /**
+   * Initialize this TokenUpdateNftsTransaction from its source TransactionBody protobuf object.
+   */
+  void initFromSourceTransactionBody();
+
+  /**
+   * Build a TokenUpdateNftsTransactionBody protobuf object from this TokenUpdateNftsTransaction object.
+   *
+   * @return A pointer to a TokenUpdateNftsTransactionBody protobuf object filled with this TokenUpdateNftsTransaction
+   * object's data.
+   */
+  [[nodiscard]] proto::TokenUpdateNftsTransactionBody* build() const;
+
+  /**
+   * The token for which to update NFTs.
+   */
+  TokenId mTokenId;
+  /**
+   * The list of serial numbers to be updated.
+   */
+  std::vector<uint64_t> mSerials;
+  /**
+   * The new metadata of the NFT(s)
+   */
+  std::vector<std::byte> mMetadata;
```

Comment:
nit: can we have spaces between a variable and the comment of the next variable?